CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # SUNDIALS Changelog
 
-## Changes to SUNDIALS in release 6.6.0
+## Changes to SUNDIALS in release 6.5.1
+
+The integrator ``ReInit`` functions have been updated to clear an existing stop
+time on reinitialization. Any codes relying on an unreached stop time persisting
```

Comment:
I don't want users to have to call an additional function to make the reinit work the way it should have worked all along.

Reply:
I understand this will change user-facing functionality, the reinit functions are actually not soooo old.  I would suggest that this is OK in a patch as it really is a bug.

Reply:
I agree the stop time should not be cleared if it has not been reached e.g., for re-initializing at a discontinuity and continuing the integration toward the stop time. We don't know if an unreached stop time is still desired after a `ReInit` call without some input from the user so I think leaving the stop time active and adding a `ClearStopTime` function is the best solution.

Reply:
Yes, you are right.  It's a reinitialization, so we should assume we are not saving much information.  I still think this should go in a patch release.